cnnlabs/cnn-hapi
# fix: update hapi and dependencies

# Description
### This is a **Breaking Change**
* This should be considered a major update `3.0.0` as it is intentionally not backwards compatible because of the major changes in hapi (ex: async/await vs callback style handlers).
* Adds support for the latest `@hapi/hapi` (18.x) and related `@hapi` libs
* Updates and fixes example files for latest hapi change
* Updates minimum supported node version to 8.12.0 to match hapi 18's minimum version
* Updated handlers to follow new `hapi` convention of using `h` instead of `reply` for 

# How to test 
If there's something I didn't test I'd be happy to review it, but I tested the following:
1. `npm run example-server`
2. curl all routes in `routes.js` and notice they all still work
  

# Questions
* I removed `localTls` temporarily, hapi no longer appears to support multiple connections at a glance, though I could be wrong. Do we need this though? 
https://github.com/hapijs/hapi/issues/3572
